setup.py

```diff
@@ -27,12 +27,12 @@
 #------------------------------------------------------------------------------
 
 from setuptools import setup
-import re
+import re, io
 
 # setup.py shall not import adal
 __version__ = re.search(
     r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]',  # It excludes inline comment too
-    open('adal/__init__.py').read()).group(1)
+    io.open('adal/__init__.py', encoding='utf8').read()).group(1)
```

Comment:
Thanks for the mentioning of `utf_8_sig`. Learn something new everyday! :-)  I'll change that in next commit, even it doesn't really matter here.